app/assets/stylesheets/custom.scss

```diff
@@ -1,6 +1,9 @@
 @import "bootstrap-sprockets";
 @import "bootstrap";
 
+
+
+
 /* universal */
 
```

Comment:
You need to add a style rule to the `<html>` tag like so:

```css
html {
  font-size: 62.5%; /* 62.5% of 16px (default font size) = 10px */
}
```

See [this article](https://www.sitepoint.com/understanding-and-using-rem-units-in-css/) for context, but essentially, the current styles force the root font size to be exactly `10px`, preventing users from customizing their own font size.